packages/tools/components-package/nps.js

```diff
@@ -45,7 +49,7 @@ const getScripts = (options) => {
 				docs: `node "${LIB}/documentation/index.js" dist/api.json`,
 			},
 			illustrations: {
-				default: `node "${LIB}/create-illustrations/index.js" ${illustrationsPath} dist/illustrations`
```

Comment:
you don't need a `default` field if it's just one command, you can pass it directly to the `illustrations` property